cmd/cloudshell_open/appfile.go

```diff
@@ -71,12 +76,13 @@ type appFile struct {
 
 	// The following are unused variables that are still silently accepted
 	// for compatibility with Heroku app.json files.
-	IgnoredDescription string   `json:"description"`
-	IgnoredKeywords    []string `json:"keywords"`
-	IgnoredLogo        string   `json:"logo"`
-	IgnoredRepository  string   `json:"repository"`
-	IgnoredWebsite     string   `json:"website"`
-	IgnoredStack       string   `json:"stack"`
+	IgnoredDescription string               `json:"description"`
+	IgnoredKeywords    []string             `json:"keywords"`
+	IgnoredLogo        string               `json:"logo"`
+	IgnoredRepository  string               `json:"repository"`
+	IgnoredWebsite     string               `json:"website"`
+	IgnoredStack       string               `json:"stack"`
+	IgnoredFormation   map[string]formation `json:"formation"`
```

Comment:
consider just saying `interface{}` here (means any type) so you don't have to define `type formation`

Reply:
oh! cool. didn't know about that.